### PR TITLE
Throw together simple form for gamelab donations

### PR DESCRIPTION
**Pull request by maxwofford**
This depends on https://github.com/hackclub/bank/pull/2611

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/donate**](https://site-g6befw2x5.hackclub.dev/donate) |
|:-:|
|<a href="https://site-g6befw2x5.hackclub.dev/donate">Screenshot of /donate</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/kUmFtpEMno5pFdYyK7aP89NWjxMtjHFQXK65jfW4dW_8xZ0DA1EXUs3ow_Ue_40rMkN3NFq5R7i62h72lkSzVw.png">(view full size)</a> |







Commit 5c25422a35e6c2eaf5453f2c056a402fb313d23f (https://site-g6befw2x5.hackclub.dev).

File: pages/donate.js [hunk 1]
```diff
@@ -428,3 +477,13 @@ export default function Donate() {
     </Box>
   )
 }
+
+export async function getServerSideProps(context) {
```
Comment (cjdenio) [inline pages/donate.js:481]: Does this need to be here? Having a `getServerSideProps` function disables [static optimization](https://nextjs.org/docs/advanced-features/automatic-static-optimization)

**Comment by zachlatta** [comment]
Please feel free to work with Max to improve the code. That would be super appreciated. I merged quickly because we have a launch tomorrow this needs to be live for

File: pages/donate.js (hunk 1 above)
Comment (sampoder) [reply pages/donate.js:481]: +1 here

**Comment by maxwofford** [comment]
Feel free to remove it @sampoder @cjdenio! This code was rapidly changing while the constraints of what this page was changed, so there's weird janky stuff in there like that.

**Pull request merged**